doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+  * LED output can be fully disabled via kConfig.
+
+* Updated
+
+  * Flash usage optimization for debug versions of samples and applications.
+  * DFU mode can now be enabled by a button press.
+  * CLI is no longer required for DFU configuration.
+  * HAP_TESTING is now configurable via kConfig.
+
+* Fixed
+
+  * An issue where RTT logs did not work with the Light Bulb multiprotocol sample with DFU on nRF52840.
+  * An issue where Nordic DFU was not compliant with HAP certification requirements.
+  * An issue where a change in KVS key naming scheme caused an error for updated devices.
+  * An issue where activating DFU caused increased power consumption.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+nRF9160: Asset Tracker v2
+-------------------------
+
+* Added:
+
+  * Support for :ref:`Bosch Software Environmental Cluster (BSEC) library <bosch_software_environmental_cluster_library>` driver.
+  * Support for Indoor Air Quality (IAQ) readings retrieved from the BME680 sensor on Thingy:91.
+    For more information, see the :ref:`asset_tracker_v2_sensor_module`.
+  * Support for QEMU x86 emulation.
+  * Support for the :ref:`lib_nrf_cloud_pgps` flash memory partition under certain conditions.
+  * Support for :ref:`QoS` library to handle multiple in-flight messages for MQTT based cloud backends such as AWS IoT, Azure IoT Hub, and nRF Cloud.
+  * Support for Lightweight Machine to Machine (LwM2M).
+  * Support for filtering updates to cloud based on LTE connection evaluation (`AT%CONEVAL`_).
+
+* Updated:
+
+  * For nRF Cloud builds, the configuration section in the shadow is now initialized during the cloud connection process.
+  * The :ref:`lib_nrf_cloud` library can now handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
+
+nRF9160: Serial LTE modem
+-------------------------
+
+* Added:
+
+  * ``#XDFUGET`` and ``#XDFURUN`` AT commands to support the cloud-to-nRF52 DFU service.
+  * Native TLS support to the HTTPS client.
+  * ``#XCMNG`` command to support the use of native TLS.
+  * ``#XSOCKETSELECT`` AT command to support multiple sockets in the Socket service.
+  * ``#XPOLL`` AT command to poll selected or all sockets for incoming data.
+
+* Updated:
+
+  * Enhanced the ``#XHTTPCREQ`` AT command for better HTTP upload and download support.
+  * Enhanced the ``#XSLEEP`` AT command to support data indication when idle.
+  * Enhanced the MQTT client to support the reception of large PUBLISH payloads.
+  * The :ref:`lib_nrf_cloud` library is now used directly instead of the Cloud API.
+
+* Fixed:
+
+  * The secondary MCUboot partition information is no longer passed to the P-GPS library if the P-GPS partition is enabled.
+  * The combined use of A-GPS and P-GPS so that ephemeris and almanac data is not requested via A-GPS, saving both power and bandwidth.
```

Comment:
```suggestion
  * The combined use of A-GPS and P-GPS so that ephemeris and almanac data is not requested through A-GPS, saving both power and bandwidth.
```

Reply:
done